src/main/resources/standardTestCases/sub_workflow_decls/sub_workflow_decls.wdl

```diff
@@ -0,0 +1,26 @@
+import "sub_workflow_decls_import.wdl" as decls
+
+workflow sub_workflow_decls {
```

Comment:
Two more thoughts:
- [ ] A passthrough input to the main WF that is required
- [ ] A passthrough input to the main WF that has a default
- [ ] A passthrough input to the main WF that has a default and is overridden